scripts/backupmanager.sh

```diff
@@ -144,7 +144,7 @@ function create_backup() {
     sleep 15
     rconcli broadcast "$(get_time) Creating backup..."
 
-    if ! tar cfz "${LOCAL_BACKUP_PATH}/${backup_file_name}" -C "${LOCAL_GAME_PATH}/" "Saved" ; then
+    if ! tar --ignore-failed-read --warning=no-file-changed --warning=no-file-removed -I zstd -cf "${LOCAL_BACKUP_PATH}/${backup_file_name}" -C "${LOCAL_GAME_PATH}/" "Saved" ; then
```

Comment:
In what situation would `--ignore-failed-read` be relevant?

I'm inclined to make the backup fail in this case. Being unable to read files seems like a valid reason to not create a backup.

Reply:
Looks to me like corrupt archive, when this goes through, im with Callum.

Reply:
As I commented [here](https://github.com/jammsen/docker-palworld-dedicated-server/pull/254#issuecomment-2041036712), I don't believe this is necessary for the PR to work.